cmd/image-processing/main.go

```diff
@@ -9,15 +9,19 @@ package main
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"log"
 	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
 
 	"github.com/google/go-containerregistry/pkg/name"
 	containerreg "github.com/google/go-containerregistry/pkg/v1"
+	buildapi "github.com/shipwright-io/build/pkg/apis/build/v1beta1"
```

Comment:
can u ensure consistency on this changes? At this point, can you then move all references of ""github.com/shipwright-io/build/pkg/apis/build/v1beta1" to buildapi in all files. A one single commit for that change is ideal.

Reply:
Done

Reply:
Just feedback: that was not a good suggestion. A large change is now an even larger PR with a lot of unrelated changes that slow down reviews.

Reply:
I suggested to make this naming consistency change in a separate PR.